keps/sig-cli/20191008-kubectl-events.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Kubectl events
+authors:
+  - "@hpandeycodeit"
+owning-sig: sig-cli
+participating-sigs:
+  - sig-cli
+reviewers:
+  - "@soltysh"
+  - "@pwittrock"
+approvers:
+  - "@soltysh"
+  - "@pwittrock"
+editor: TBD
+creation-date: 2019-10-08
+last-updated: 2019-10-08
+status: provisional
+see-also:
+  - n/a
+replaces:
+  - 
+superseded-by:
+  -
+---
+
+# kubectl events
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Long standing issues](#long-standing-issues)
+  - [Goals](#goals)
+- [Proposal](#proposal)
+  - [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+- [Graduation Criteria](#graduation-criteria)
+<!-- /toc -->
+
+## Summary
+
+Presently, `kubectl get events` has some limitations. It cannot be extended to met the increasing user needs to 
+support more functionality without impacting the `kubectl get`. 
+
+For eg: Any modification to `--watch` functionality for `events` will also change the `--watch` for `kubectl get` since the `events` is dependent of `kubectl get`
+
+Some of the requested features for events include: 
+
+1. Extendeb behaviour for `--watch` 
```

Comment:
"Extended"

Reply:
Thanks for pointing it out :) . Fixed these